### ISSUE TITLE
map=0

### ISSUE THREAD

**Issue by yeyun122** [xiaolongren969/SNN-YOLOv3#1]
Hello, I would like to ask, why do I train for several epochs and the accuracy is always 0

**Comment by xiaolongren969** [comment]
you need download yolov3 pretrained model
